modules/snphost/mkosi.extra/usr/local/lib/systemd/system/snphost-ok.service

```diff
@@ -1,8 +1,8 @@
 [Unit]
 Description=Run snphost ok to  make sure host is correctly set-up for SNP functionalities.
 DefaultDependencies=no
-After=network-online.target
-Wants=network-online.target
+After=network-online.target beacon-boot.service
+Wants=network-online.target beacon-boot.service
```

Comment:
Does this actually need network-online? `beacon-boot` already requires `network-online`.